apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/oppgaveModal/OmgjoerVedtakModal.tsx

```diff
@@ -70,11 +79,24 @@ export function OmgjoerVedtakModal({ oppgave }: { oppgave: OppgaveDTO }) {
       return
     }
 
-    if (erBehandlingVedtakOmgjoering(klage) && isInitial(opprettRevurderingStatus)) {
+    const handling = finnOmgjoeringsHandlingForKlage(klage)
+
+    if (handling === OmgjoerHandling.REVURDERING) {
       opprettRevurdering({
         oppgaveId: oppgave.id,
         sakId: oppgave.sakId,
       })
+    } else if (handling === OmgjoerHandling.OMGJOERING_FOERSTEGANGSBEHANDLING) {
+      opprettOmgjoeringFoerstegangsbehandlingApi({
+        sakId: oppgave.sakId,
+        omgjoeringRequest: {
+          skalKopiere: true,
+          erSluttbehandlingUtland: false,
+          omgjoeringsOppgaveId: oppgave.id,
+        },
+      })
+    } else {
+      // TODO: exception?
```

Comment:
Jeg tenker at vi enten her kan sette en feilmelding som vises i modalen, eller så kan vi alternativt si at den eksisterende meldingen er god nok, og vi kan la være å vise knappen for å opprette omgjøringen

Reply:
Vil ikke denne slå til? 

```
{mapApiResult(
...
(error) => (
  <ApiErrorAlert>{error.detail}</ApiErrorAlert>
),
...
)}
```

Så vi trenger egentlig ingen videre håndtering i front-end ettersom vi returnerer feil fra backend?

Reply:
Den vil ikke slå til, siden vi ikke faktisk gjør noe kall (når du trykker på knappen vil funksjonen kjøres, men hvis handling er IKKE_STOETTET så kaller vi aldri backend)